src/commands/sentinel-pending-scripts.json

```diff
@@ -10,6 +10,43 @@
             "ADMIN",
             "SENTINEL",
             "ONLY_SENTINEL"
-        ]
+        ],
+        "reply_schema": {
+            "type": "array",
+            "description": "List of pending scripts.",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "argv": {
+                        "type": "array",
+                        "description": "Script arguments.",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "flags": {
+                        "type": "string",
+                        "description": "Script flags."
+                    },
+                    "pid": {
```

Comment:
pid should be an integer value

Reply:
```c
addReplyBulkCString(c,"pid");
addReplyBulkLongLong(c,sj->pid);
```
this look like a string

Reply:
i think you could reference acl-log json, entry-id, timestamp-created and timestamp-last-updated, three of them are "type": "integer", so I think update pid to integer is better

Reply:
acl log use `addReplyLongLong`, so it should indeed be described as int.
and the pid use `addReplyBulkLongLong` , so it should be a string i think
```
            addReplyBulkCString(c, "entry-id");
            addReplyLongLong(c, le->entry_id);
            addReplyBulkCString(c, "timestamp-created");
            addReplyLongLong(c, le->timestamp_created);
            addReplyBulkCString(c, "timestamp-last-updated");
            addReplyLongLong(c, le->ctime);
```

Reply:
this is similar to the problem with sentinel-info-cache, which is that the code changes to adapt sentinel to RESP3 weren't done completely.

and again, either we do a breaking change, or we keep it a bulk string, and keep the schema a string.
we can add a comment saying it's an integer, but i don't think that's necessary since it's a PID.